backend/frontend/templates/frontend/base_logged_in.html

```diff
@@ -1,6 +1,7 @@
 {% extends 'frontend/base.html' %}
 {% load static %}
 {% block main_head %}
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet" />
```

Comment:
frontend/templates/frontend/base.html includes the line:

```
    <script src="https://cdn.tailwindcss.com"></script>
```

And this template extends base.

Why are you including tailwind again? And why should different parts of the website run different versions of tailwind?